client/src/portal/item.rs

```diff
@@ -27,7 +27,7 @@ impl Item {
     pub(crate) fn new(
         label: impl ToString,
         attributes: &impl AsAttributes,
-        secret: impl AsRef<[u8]>,
+        secret: impl Into<Secret>,
```

Comment:
Again related to into:

I have seen many calls to into. I don't really understand this yet. Can't we try to use From here and other places instead of Into.

Reply:
Same as above :)